docs/notebooks/channel.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        plt.ylabel(r"transmittance", fontsize=15)
</pre><p>There are no backslashes so the <code>r</code> can be dropped in the string </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HinodeXRT/xrtpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>